Automattic/wp-calypso
# Receipts: Show VAT Vendor info for all eligible receipts

Currently the receipt view shows VAT vendor information only if the user has saved VAT details. However, now that we collect price-exclusive VAT, we should show the vendor info no matter what if the purchase is in a country that would have that info. In addition, the VAT vendor info shown should be localized and have information specific to the country of the receipt.

## Proposed Changes

This PR refactors the receipt page to always show the vat vendor info if the receipt is for a country for which we charge VAT.

Requires D102507-code to attach country codes to receipts.

This is part of https://github.com/Automattic/payments-shilling/issues/1403

## Screenshots

Before, without VAT id saved:

<img width="1056" alt="Screenshot 2023-02-23 at 12 48 27 PM" src="https://user-images.githubusercontent.com/2036909/220988494-1f75c616-aeed-4f25-8235-fde97211b99e.png">

After, without VAT id saved:

<img width="1056" alt="Screenshot 2023-02-23 at 12 48 35 PM" src="https://user-images.githubusercontent.com/2036909/220988509-be38d3c6-8a91-4ee3-adc3-d01bdcfb9c53.png">

## Testing Instructions

To test this, you'll need to use an account that has had purchases in the EU.

- Visit `/me/purchases` and click on a purchase to view its receipt.
- Verify that you see the VAT Vendor information displayed.
